general/solomiia_tymoshchuk/lecture3/_4v2/ReporterBase.java

```diff
@@ -0,0 +1,38 @@
+package general.solomiia_tymoshchuk.lecture3._4v2;
+
+/**
+ * Did the second version for practising abstract classes use; and actually, as you showed analogue of use in lesson as example.
```

Comment:
actually, the task was planned to be implemented in a similar way as your second version 😄

Reply:
okay) Thank you for the comments, will update the work.